lib360dataquality/cove/threesixtygiving.py

```diff
@@ -1239,6 +1308,12 @@ def process(self, grant, path_prefix):
                 if not org_id:
                     continue
                 id_location = "{}/{}/{}/id".format(path_prefix, org_type, num)
+
+                if "\n" in org_id:
+                    self.failed = True
+                    self.json_locations.append(id_location)
+                    self.count += 1
+
```

Comment:
Why is this check repeated here? Would this not be caught by the `OrganizationIdInvalidChars` test?

Reply:
Ah thanks. That was from an earlier plan to have the invalid chars check  inside an existing check then we changed our mind. Will remove.